### PR TITLE
feat(Home): renamed components name and refactored some codes.

### PR DESCRIPTION
**Pull request by ravinalamada**
Hi team,

This is role-selectionOnboardingOne from Nivia team. We did some refactors in this branch. Can you review it please? We'd like to get any comments in order to get it improved.

Refs: #41k8dk

Thanks in advance,

### CODE REVIEW

File: src/components/RoleSelectOption/RoleSelectOption.tsx [hunk 1]
```diff
@@ -36,70 +37,43 @@ const optionWrapperStyles = css`
...
-          className={
-            label === 'Buyer'
-              ? 'dark-polygon-wrapper-bg'
-              : 'white-polygon-wrapper-bg'
-          }>
-          <img src={src} alt={alt} />
+      <RoleSelectBackground
+        onMouseEnter={() => setIsHovered(true)}
+        onMouseLeave={() => setIsHovered(false)}>
+        <div className='polygon-bg-image'>
```
Comment (andrzej-onja) [inline src/components/RoleSelectOption/RoleSelectOption.tsx]: We are using styled components so no class names assignments.
Please extract div into separate styled one

File: src/components/Home/Home.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+import { DescriptionOffer } from '../DescriptionOffer/descriptionOffer'
+import { Option } from '../RoleSelectOption/RoleSelectOption'
+import userIcon from '../../stories/assets/user.svg'
+import userIconHovered from '../../stories/assets/userNotHovered.svg'
+import briefcaseIcon from '../../stories/assets/briefcase.svg'
+import briefcaseIconHovered from '../../stories/assets/briefcaseNotHovered.svg'
+import { Login } from '../Login/Login'
+import { LeftSide } from '../LeftSide/LeftSide'
+
+export const Home: React.FC<Props> = ({ footerLinks }) => {
```
Comment (andrzej-onja) [inline src/components/Home/Home.tsx]: Create a new folder Pages and put this component inside. Also rename it to HomePage

**Review by andrzej-onja — CHANGES_REQUESTED** [review]

File: src/App.tsx [hunk 3]
```diff
@@ -1,39 +1,11 @@
...
-// import { useSelector, useDispatch } from 'react-redux'
-import {
-  adsSelector,
-  decrement,
-  fetchAds,
-  incrementByAmount,
-} from './redux/slices/adsSlice'
-import { /*useAppSelector,*/ useAppDispatch } from './redux/hooks'
-import { useSelector } from 'react-redux'
+import { Home } from './Pages/HomePage.story'
```
Comment (andrzej-onja) [inline src/App.tsx]: HomePage.story ?

File: src/components/RoleSelectOption/RoleSelectOption.tsx [hunk 4]
```diff
@@ -36,124 +37,87 @@ const optionWrapperStyles = css`
   border-radius: 6px;
 `
 
-export const Option: React.FC<OptionProps> = ({ label, text, src, alt }) => {
+export const Option: React.FC<OptionProps> = ({
```
Comment (andrzej-onja) [inline src/components/RoleSelectOption/RoleSelectOption.tsx]: Rename to RoleSelectOption

Comment (andrzej-onja) [reply src/components/RoleSelectOption/RoleSelectOption.tsx]: add onClick prop

File: src/components/RoleSelectionOnboardingOne/RoleSelectionOnBoardingOne.tsx [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+import { DescriptionOffer } from '../DescriptionOffer/descriptionOffer'
+import { Option } from '../RoleSelectOption/RoleSelectOption'
+import userIcon from '../../stories/assets/user.svg'
+import userIconHovered from '../../stories/assets/userNotHovered.svg'
+import briefcaseIcon from '../../stories/assets/briefcase.svg'
+import briefcaseIconHovered from '../../stories/assets/briefcaseNotHovered.svg'
+import { Login } from '../Login/Login'
+import { LeftSide } from '../LeftSide/LeftSide'
+
+export const RoleSelectOnboardOne = () => {
```
Comment (andrzej-onja) [inline src/components/RoleSelectionOnboardingOne/RoleSelectionOnBoardingOne.tsx]: Rename to  just RoleSelectOnboard

File: src/components/RoleSelectionOnboardingOne/RoleSelectionOnBoardingOneStyle.tsx [hunk 6]
```diff
@@ -1,24 +1,27 @@
-import styled from "styled-components"
-
+import styled from 'styled-components'
```
Comment (andrzej-onja) [inline src/components/RoleSelectionOnboardingOne/RoleSelectionOnBoardingOneStyle.tsx]: rename RoleSelectOnboardStyle

File: src/components/RoleSelectionOnboard/RoleSelectionOnBoard.tsx [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+import { DescriptionOffer } from '../DescriptionOffer/descriptionOffer'
+import { Option } from '../RoleSelectOption/RoleSelectOption'
+import userIcon from '../../stories/assets/user.svg'
+import userIconHovered from '../../stories/assets/userNotHovered.svg'
+import briefcaseIcon from '../../stories/assets/briefcase.svg'
+import briefcaseIconHovered from '../../stories/assets/briefcaseNotHovered.svg'
+import { Login } from '../Login/Login'
+import { LeftSide } from '../LeftSide/LeftSide'
+
+export const RoleSelectOnboard = () => {
```
Comment (andrzej-onja) [inline src/components/RoleSelectionOnboard/RoleSelectionOnBoard.tsx:18]: useState,  [selectedRole,setSelectedRole] - called from RoleSelectOption wit selected role string

**Review by andrzej-onja — APPROVED** [review]

**Review by vakodrazan — APPROVED** [review]

**Review by bakozetra — APPROVED** [review]

**Pull request merged**